src/xbps/package-management.md

```diff
@@ -111,3 +111,22 @@ To get a list of all installed packages, without their version:
 ```
 $ xbps-query -l | awk '{ print $2 }' | xargs -n1 xbps-uhelper getpkgname | fmt
 ```
+
+## Ignoring Packages
+
+Sometimes you may wish to remove a package whose functionality is being provided
+by another package, but will be unable to do so due to dependency issues. For
```

Comment:
"functionality is being provided by another package" is not the only case of dependency removal being valid. I'm not sure we want to mention other cases, though, because we don't want to support people stripping down their systems unnecessarily.

Reply:
Agreed on both points. So leave it as-is?

Reply:
I think so, yeah.

Reply:
Thanks. i've amended the PR to address your other comment. Ok to merge?

Reply:
Ok! Thanks c